pkg/api/persistentvolume/util.go

```diff
@@ -42,7 +42,13 @@ func VisitPVSecretNames(pv *api.PersistentVolume, visitor Visitor) bool {
 		}
 		return true
 	case source.CephFS != nil:
-		if source.CephFS.SecretRef != nil && !visitor(getClaimRefNamespace(pv), source.CephFS.SecretRef.Name) {
```

Comment:
only do the `if source.CephFS.SecretRef != nil` check once, move all the other logic inside an `if` block